source/getting-started/first-app.md

```diff
@@ -7,28 +6,29 @@ ember new my-new-app
 
 This will create a new `my-new-app` directory with your new Ember app inside.
 
-Once the generation process finishes, launch the Ember CLI development server:
+Once the process finishes, launch the Ember development server:
```

Comment:
(nvm, I misread and on https://github.com/emberjs/guides/pull/556/files#diff-bb19732339002c2a1d24dda65b43200bR18 it is "Ember CLI" already)